CMakeLists.txt

```diff
@@ -621,26 +621,14 @@ if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.29)
   set(CMAKE_SKIP_TEST_ALL_DEPENDENCY FALSE)
 endif()
 
-# TODO: The `CMAKE_SKIP_BUILD_RPATH` variable setting can be deleted
-#       in the future after reordering Guix script commands to
-#       perform binary checks after the installation step.
-# Relevant discussions:
-# - https://github.com/hebasto/bitcoin/pull/236#issuecomment-2183120953
-# - https://github.com/bitcoin/bitcoin/pull/30312#issuecomment-2191235833
-# NetBSD always requires runtime paths to be set for executables.
-if(CMAKE_SYSTEM_NAME STREQUAL "NetBSD")
-  set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
-else()
-  set(CMAKE_SKIP_BUILD_RPATH TRUE)
-endif()
+set(CMAKE_SKIP_INSTALL_RPATH TRUE)
```

Comment:
Is there any reason to keep this line?

Reply:
I had thought it was still needed, but on closer analysis you're correct, thanks for catching that. Guix still passes `-DCMAKE_SKIP_RPATH=TRUE`, so the post-install security and symbol checks remain on the installed binaries without hard-coding it in the top-level CMake file, and NetBSD keeps its RPATH behavior. I've amended the commit.